ts/src/profiler.ts

```diff
@@ -245,13 +245,18 @@ export class Profiler extends ServiceObject {
   config: ProfilerConfig;
 
   constructor(config: ProfilerConfig) {
-    config = util.normalizeArguments(null, config) as ProfilerConfig;
-    const serviceConfig = {
+    config = config || {} as ProfilerConfig;
+    const serviceConfig: ServiceConfig = {
       baseUrl: config.baseApiUrl,
       scopes: [SCOPE],
       packageJson: pjson,
+      requestModule: request,
     };
-    super({parent: new Service(serviceConfig, config), baseUrl: '/'});
+    super({
+      parent: new Service(serviceConfig, config),
+      baseUrl: '/',
+      requestModule: request
```

Comment:
Q: Not yet time to use [teeny-request](https://npm.im/teeny-request) here?

Reply:
Babysteps!